message/job.py

```diff
@@ -35,12 +42,14 @@ class Message:
     return_message = attr.ib(default=None)
     retry_in = attr.ib(default=None)
 
-    def serialize(self):
+    # Note: added indent to allow pretty-printing
```

Comment:
if defaults to None this is fine, but just be sure to add it in the param docstring and give explanation